verify_rpms/rpm_verifier.py

```diff
@@ -169,19 +177,14 @@ def main(
 ) -> None:
     """Verify RPMs are signed"""
 
-    container_images = parse_image_input(img_input)
-
     processor = ImageProcessor(workdir=workdir)
-    with ThreadPoolExecutor() as executor:
-        processed_images: Iterable[ProcessedImage] = executor.map(
-            processor, container_images
-        )
+    processed_image = processor(img=img_input)
 
-    failures_occurred, output = generate_output(list(processed_images))
+    failures_occurred, output = generate_output(processed_image=processed_image)
     if failures_occurred:
-        status_path.write_text("ERROR")
+        status_path.write_text("ERROR")  # pylint: disable=unspecified-encoding
```

Comment:
Did it start to complain about unspecified-encoding now? Is it failing on that?

Reply:
Strange, I got some errors with pylint, I do not get them anymore.
I will remove it